kairon/shared/data/data_models.py

```
@@ -1268,11 +1268,14 @@ def check(cls, values):
 
         if non_editable:
             non_editable_set = set(non_editable)
+            data_keys = set(data.keys())
             if not non_editable_set.issubset(data_keys):
                 raise ValueError("is_non_editable contains keys that are not present in data")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)